app/build.gradle

```diff
@@ -17,6 +17,7 @@ ext {
     espUIAVer = "2.1.2"
     firebaseVer = "11.0.0"
     glideVer = "4.0.0-RC0"
+    playServicesVer = "11.0.0"
```

Comment:
similar for firebase can be done. you can create a separate issue if required or complete in this PR